osu.Game/Beatmaps/Events/EventInfo.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using System.Collections.Generic;
+using System.Linq;
+
+namespace osu.Game.Beatmaps.Events
+{
+    public class EventInfo

```

Comment:
still don't think we want this. just make a list of breaks inside Beatmap for now